Volodymyr_Kuksa/2/Allergies.py

```diff
@@ -0,0 +1,26 @@
+def determine_allergies(allergy_score):
+    """
+    Return list of allergies, determined by allergy_score.
+
+    :param allergy_score: sum of all allergy scores.
+    :type allergy_score: int
+
+    :return: list of allergy names.
+    :rtype: list.
+    """
+    allergy_names = ['eggs', 'peanuts', 'shellfish', 'strawberries', 'tomatoes', 'chocolate', 'pollen', 'cats']
+    allergy_values = [1, 2, 4, 8, 16, 32, 64, 128]
+
+    result = list()
```

Comment:
It's better to use empty square brackets - result = [].

Reply:
These are minor changes, may I not change it for these tasks?  
(I will use `[]` for next tasks)

Reply:
Yes. You can skip minor changes till the next batch if your current batch is approved and merged.